### DISCUSSION TITLE
코드 한 줄의 최대 길이 제한이 필요한 이유

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#121] [General]
## 코드 한 줄의 최대 길이 제한이 필요한 이유

[\[Minimalist Programmer\] 코드 한 줄의 최대 길이를 80자로 해야 하는 이유](https://sangheon.com/%EC%BD%94%EB%93%9C-%ED%95%9C-%EC%A4%84%EC%9D%98-%EC%B5%9C%EB%8C%80-%EA%B8%B8%EC%9D%B4%EB%A5%BC-80%EC%9E%90%EB%A1%9C-%ED%95%B4%EC%95%BC%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0/)

최근에 코드 한 줄의 최대 길이 제한에 대한 언급을 여러 자료에서 봤었습니다.    

모바일 장치의 가로 뷰 & 세로 뷰까지 고려하여   
최대한 가독성이 좋도록 코드 한 줄의 길이를 제한해야 한다더군요.

**Comment by Lcat12345** [comment]
잘 읽었습니다
